dotCMS/src/main/java/com/dotmarketing/util/ImportExportUtil.java

```diff
@@ -1114,21 +1112,13 @@ private void doXMLFileImport(File f, PrintWriter out, ObjectFilter filter)throws
                 logsMappers = true;
             }else if(_className.equals("Portletpreferences")){
                 portletpreferences = true;
-            }else if(_className.equals("Pollschoice")){
-                pollschoice = true;
             }else if(_className.equals("Address")){
                 address = true;
-            }else if(_className.equals("Pollsdisplay")){
-                pollsdisplay = true;
-            }else if(_className.equals("Pollsquestion")){
-                pollsquestion = true;
-            }else if(_className.equals("Pollsvote")){
-                pollsvote = true;
             }else{
                 try{
                     _importClass = Class.forName(_className);
                 }catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)